games/devtest/mods/unittests/metadata.lua

```diff
@@ -55,6 +55,11 @@ local function test_metadata(meta)
 	assert(meta:get_float("a") == 1.0)
 	assert(meta:get_int("e") == 0)
 	assert(meta:get_float("e") == 0.0)
+	
+	meta:set_bool("j", true)
+	assert(meta:get_bool("j") == true)
+	meta:set_bool("j", false)
+	assert(meta:get_bool("j") == false)
```

Comment:
should add checks for the errors too, you can use `pcall` to catch Lua errors.

Reply:
Except none of these error so there’s no need.